mu4/userscores/qml/MuseScore/UserScores/ChooseTemplatePage.qml

```diff
@@ -0,0 +1,98 @@
+import QtQuick 2.7
+import QtQuick.Layouts 1.3
+
+import MuseScore.UiComponents 1.0
+import MuseScore.Ui 1.0
+import MuseScore.UserScores 1.0
+
+Item {
+    property alias selectedTemplatePath: model.currentTemplatePath
+
+    TemplatesModel {
+        id: model
+
+        onCurrentTemplateChanged: {
+            templatePreview.load(model.currentTemplatePath)
+        }
+    }
+
+    Component.onCompleted: {
+        model.load()
+    }
+
+    RowLayout {
+        anchors.fill: parent
+
+        spacing: 12
+
+        TitleListView {
+            Layout.fillHeight: true
+            Layout.preferredWidth: parent.width / 4
+
+            listTitle: qsTrc("userscores", "Category")
+            model: model.categoriesTitles
+            searchEnabled: false
+            boldFont: true
+
+            onTitleClicked: {
+                model.setCurrentCategory(index)
+            }
+        }
+
+        SeparatorLine { orientation: Qt.Vertical }
+
+        TitleListView {
+            Layout.fillHeight: true
+            Layout.preferredWidth: parent.width / 4
+
+            listTitle: qsTrc("userscores", "Template")
+            model: model.templatesTitles
+
+            onTitleClicked: {
+                model.setCurrentTemplate(index)
+            }
+
+            onSearchTextChanged: {
+                model.setSearchText(searchText)
+            }
+        }
+
+        SeparatorLine { orientation: Qt.Vertical }
+
+        Item { Layout.preferredWidth: 30 }
```

Comment:
Usually, you don't need any spacers in Layout component, because there are Layout.leftMargin, Layout.rightMargin, Layout.alignment and so on

So why do you need it here?

Reply:
This is a constant spacer. After changing the dialog size, the categories/templates list and the template preview will be proportionally resized, but this spacer must save its size

![изображение](https://user-images.githubusercontent.com/67867444/93193567-4e806f00-f747-11ea-87ed-3bdea19d76cd.png)

Reply:
OK, so you just need to set a leftMargin for preview
`TemplatePreview {
     id: templatePreview
     ....
     Layout.leftMargin: 30
`

Reply:
fixed